velox/dwio/parquet/reader/PageReader.h

```diff
@@ -68,8 +68,8 @@ class PageReader {
         chunkSize_(chunkSize),
         nullConcatenation_(pool_) {}
 
-  common::CompressionKind ThriftCodecToCompressionKind(
-      thrift::CompressionCodec::type codec) const {
+  static common::CompressionKind ThriftCodecToCompressionKind(
```

Comment:
This function is only used in PageReader.cpp so it doesn't need to have global scope which static functions do. Also it uses member codec_. Let's make it a private member function without parameters and move it to the private section.

Reply:
@yingsu00 Made the change. Please review.